Feature/remlines dragon age

rayo329 opened the pull request.

Dragonage13 committed:
RemLines from lines 1 to 1004

rayo329 committed:
fix: mistake

rayo329 committed:
Merge branch 'develop' into feature/Remlines-dragonAge

Dragonage13 committed:
Revisado lineas de 1 a 1004

rayo329 commented on RemLines.json:
```diff
@@ -40,7 +40,7 @@
 "karryn_sight_exp0_4":{
 "en":["\\fb\\C[5](¿Qué pasa con este chico?......",
-	"\\fb\\C[5]Comparado con Su Majestad Imperial,",
+	"\\fb\\C[5]Comparado con su Majestad Imperial,",
```
Su Majestad Imperial

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 "karryn_sight_exp2_1":{
 "en":["\\fb\\C[5]\"Fufufu......",
-	"\\fb\\C[5]Continúe, eche un buen vistazo a mi impresionante cuerpo.\""],
+	"\\fb\\C[5]Vamos continua, echa un buen vistazo a mi impresionante cuerpo.\""],
```
Vamos, continúa,

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 "karryn_sight_exp2_2":{
 "en":["\\fb\\C[5]\"Oh, ¿quieres echar un vistazo a mi cuerpo?",
-	"\\fb\\C[5]Aquí, cómete con los ojos.\""],
+	"\\fb\\C[5]Aquí, ven y hecha una mirada.\""],
```
Ese echar es sin "h". 
Valorar si suena más natural cambiar por "Acércate y échame un ojo"

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 "karryn_sight_exp2_4":{
 "en":["\\fb\\C[5]\"Me gusta la forma en que me miras......",
-	"\\fb\\C[5]Continúe, siga mirando.\""],
+	"\\fb\\C[5]Contina, sigue mirando.\""],
```
Contina -> Continúa

En mi opinión, mejor cambiar por "Vamos, sigue mirando"

TrueEnjoyer replied:
Pienso en algo mas en el sentido de que sea deseada
"Continúa, sigue mirándome."
 pero ustedes tiene la decisión final

rayo329 replied:
También lo veo bien

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 
 "karryn_sight_exp2_5":{
-"en":["\\fb\\C[5]\"Estás fantaseando con follarme ahora mismo, ¿no?\""],
+"en":["\\fb\\C[5]\"Estás fantaseando con follarme ahora mismo, ¿no es así?\""],
```
Mayúscula tras signo de interrogación

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 
 "karryn_sight_exp3_1":{
-"en":["\\fb\\C[5]\"Jajaja♥ ¡Máaaaaaaaaas! Violame con tus ojos!♥\""],
+"en":["\\fb\\C[5]\"Haa haa♥ ¡Máaaaaaaaaas! ¡Viólame con tus ojos!♥\""],
```
Esos "haa" son jadeos. Es mejor cambiarlos por "Aah aah".

En mi opinión, es más natural "Viólame con tu/la mirada"

rayo329 commented on RemLines.json:
```diff
@@ -67,49 +67,49 @@
 
 "karryn_sight_exp3_4":{
-"en":["\\fb\\C[5]\"Simplemente amo♥ a un hombre con una libido desenfrenada.♥\""],
+"en":["\\fb\\C[5]\"Simplemente amo♥ a un hombre con una libido desenfrenado.♥\""],
```
libido es femenino. "desenfrenada"

rayo329 commented on RemLines.json:
```diff
@@ -125,7 +125,7 @@
 
 "karryn_sight_bukkake_exp1_2":{
-"en":["\\fb\\C[5](Definitivamente está mirando el semen que me salpicó.......)"],
+"en":["\\fb\\C[5](Él definitivamente está mirando el semen que tengo encima.......)"],
```
Opcional: Omitir este "Él" para sonar más natural

rayo329 commented on RemLines.json:
```diff
@@ -140,7 +140,7 @@
 
 "karryn_sight_bukkake_exp2_1":{
-"en":["\\fb\\C[5]\"Fufufu… ¿y bien? Tengo un aspecto desagradable, ¿no?\""],
+"en":["\\fb\\C[5]\"Fufufu… ¿y bien? Me veo asquerosa, ¿no?\""],
```
Mayúsculas tras signos de interrogación

rayo329 commented on RemLines.json:
```diff
@@ -160,12 +160,12 @@
 
 "karryn_sight_bukkake_exp2_5":{
-"en":["\\fb\\C[5]\"Tú también quieres que me corras, ¿verdad?\""],
+"en":["\\fb\\C[5]\"Tú también quieres echar tu semen en mí, ¿cierto?\""],
```
Mayúscula tras signo de interrogación.

Valorar cambiar por una expresión más natural, EJ: "Tú también quieres echarme tu semen, ¿Verdad?"

rayo329 commented on RemLines.json:
```diff
@@ -175,17 +175,17 @@
 
 "karryn_sight_bukkake_exp3_3":{
-"en":["\\fb\\C[5]\"Échame más jugo de hombre.♥\""],
+"en":["\\fb\\C[5]\"Échame más jugo de hombre en mí.♥\""],
```
Jugo de hombre -> Leche

"Echame más leche" 
Valorar omitir "en mí" para sonar más natural

rayo329 commented on RemLines.json:
```diff
@@ -175,17 +175,17 @@
 
 "karryn_sight_bukkake_exp3_4":{
-"en":["\\fb\\C[5]\"Jajaja♥ Me estoy poniendo muy cachonda ahora mismo.♥\""],
+"en":["\\fb\\C[5]\"Haa haa♥ Me estoy poniendo muy cachonda ahora mismo.♥\""],
```
Jadeos
"Aah aah"

rayo329 commented on RemLines.json:
```diff
@@ -175,17 +175,17 @@
 
 "karryn_sight_bukkake_exp3_5":{
-"en":["\\fb\\C[5]\"No te limites a mirar, báñame con tu semen también.♥\""],
+"en":["\\fb\\C[5]\"No te límites a mirar, báñame con tu semen también.♥\""],
```
Esta tilde en limites está mal

rayo329 commented on RemLines.json:
```diff
@@ -196,7 +196,7 @@
 
 "karryn_sight_mouth_exp0_2":{
-"en":["\\fb\\C[5](¿Qué pasa con este hombre? Sigue mirándome la boca....)"],
+"en":["\\fb\\C[5](¿Qué pasa con este hombre? Sigue mirándome la boca…)"],
```
Mantén los puntos finales originales, se han debido cambiar de formato al copiar/pegar

rayo329 commented on RemLines.json:
```diff
@@ -211,12 +211,12 @@
 
 "karryn_sight_mouth_exp1_2":{
-"en":["\\fb\\C[5]\"¡¿Parece que besaría a un pedazo de basura como tú?!\""],
+"en":["\\fb\\C[5]\"¡¿Te parece que besaría a un pedazo de basura como tú?!\""],
```
Valorar cambiar a frase más natural 

"¡¿Tú te crees que besaría a..."

rayo329 commented on RemLines.json:
```diff
@@ -231,12 +231,12 @@
 
 "karryn_sight_mouth_exp2_2":{
-"en":["\\fb\\C[5](¿Quiere que lo bese...?)"],
+"en":["\\fb\\C[5](¿Él quiere que lo bese...?)"],
```
Valorar omitir este él para sonar más natural

rayo329 commented on RemLines.json:
```diff
@@ -251,7 +251,7 @@
 
 "karryn_sight_mouth_exp3_1":{
-"en":["\\fb\\C[5]\"Oh, ¿quieres que beba tu semen también?\""],
+"en":["\\fb\\C[5]\"Oh, ¿quieres que me beba tu semen también?\""],
```
Mayúscula tras signo de interrogación

rayo329 commented on RemLines.json:
```diff
@@ -231,12 +231,12 @@
 
 "karryn_sight_mouth_exp2_3":{
-"en":["\\fb\\C[5]\"Apuesto a que tú también quieres chuparme la lengua.\""],
+"en":["\\fb\\C[5]\"Apuesto a que tú también quieres chupar mi lengua.\""],
```
Estaba bien antes

rayo329 commented on RemLines.json:
```diff
@@ -262,13 +262,13 @@
 "karryn_sight_mouth_exp3_3":{
-"en":["\\fb\\C[5]\"Sí, te gustaría que usara estos",
-	"\\fb\\C[5]labios en tu polla, ¿no?♥\""],
+"en":["\\fb\\C[5]\"Sí, te gustaría que usara estos labios",
```
Omitir el "Si" del principio para sonar más natural.

Mayúscula tras "¿"

rayo329 commented on RemLines.json:
```diff
@@ -351,12 +351,12 @@
 
 "karryn_sight_boobs_exp3_3":{
-"en":["\\fb\\C[5]\"Fufu, how adorable.♥\""],
+"en":["\\fb\\C[5]\"Fufu, que adorable.♥\""],
```
Aunque este enunciado esté escrito sin signos de exclamación tiene sentido exclamativo. El «qué» que lo introduce es un exclamativo que se pronuncia tónico y se escribe con tilde.

rayo329 commented on RemLines.json:
```diff
@@ -351,12 +351,12 @@
 
 "karryn_sight_boobs_exp3_4":{
-"en":["\\fb\\C[5]\"Eres un chico con tetas, ¿no?♥\""],
+"en":["\\fb\\C[5]\"Eres un chico que le gustan las tetas, ¿no?♥\""],
```
Mayúscula tras "¿"

Valorar cambiar a "Eres un chico al que le gustan las tetas ---"
para sonar más natural

rayo329 commented on RemLines.json:
```diff
@@ -403,7 +403,7 @@
 "karryn_sight_boobs_bokki_exp2_3":{
 "en":["\\fb\\C[5]\"Hng......",
-	"\\fb\\C[5]Ellos se pondrán más duros si sigues comiendome con los ojos.......\""],
+	"\\fb\\C[5]Ellos se pondrán más duros si sigues mirándolos fijamente.......\""],
```
Está hablando de sus pezones, quitar "ellos"

"Se me van a poner más duros si sigues mirándolos fijamente"

rayo329 commented on RemLines.json:
```diff
@@ -439,7 +439,7 @@
 
 "karryn_sight_boobs_bokki_exp3_5":{
-"en":["\\fb\\C[5]\"Ahnn ♥ La forma en que estás comiendo mis pezones ♥ ​​es tan ~ lascivo. ♥\""],
+"en":["\\fb\\C[5]\"Ahnn ♥ La forma en que estás mirando mis pezones ♥ es tan ~ lascivo. ♥\""],
```
lascivo -> lasciva

el sujeto es "**la forma** en que estás mirando mis pezones"

rayo329 commented on RemLines.json:
```diff
@@ -549,7 +549,7 @@
 
 "karryn_sight_pussy_exp1_2":{
-"en":["\\fb\\C[5]\"¿Crees que podrás abrir mis piernas con tus ojos?\""],
+"en":["\\fb\\C[5]\"¿Que, crees que podrás abrir mis piernas con tus ojos?\""],
```
tilde en Qué exclamativo.

rayo329 commented on RemLines.json:
```diff
@@ -569,7 +569,7 @@
 
 "karryn_sight_pussy_exp2_3":{
-"en":["\\fb\\C[5]\"Se está preparando para follarme...\""],
+"en":["\\fb\\C[5]\"Aun no estoy mojada.\""],
```
"Aún" es tónico, lleva tilde
El original es "I'm not wet enough yet"

Valorar cambiar a "Aún no estoy tan mojada"

rayo329 commented on RemLines.json:
```diff
@@ -589,7 +589,7 @@
 
 "karryn_sight_pussy_wet_exp0_1":{
-"en":["\\fb\\C[5](Me vas a mojar si sigues mirándome con los ojos así.♥)"],
+"en":["\\fb\\C[5](Oh no.…. Estoy mojada ahora mismo.♥)"],
```
Creo que aquí se hann vuelto a fastidiar el formato de los puntos suspensivos, copialos de la frase de abajo

rayo329 commented on RemLines.json:
```diff
@@ -645,7 +645,7 @@
 
 "karryn_sight_pussy_nakadashi_exp1_2":{
-"en":["\\fb\\C[5]\"Grr... No puedo enfrentar a Su Majestad Imperial así...\""],
+"en":["\\fb\\C[5]\"Grr... No puedo mostrarme frente a su Majestad Imperial así...\""],
```
Su Majestad Imperial

mayúscula en las 3

rayo329 commented on RemLines.json:
```diff
@@ -655,7 +655,7 @@
 
 "karryn_sight_pussy_nakadashi_exp2_1":{
-"en":["\\fb\\C[5](Él me está mirando dagas......)"],
+"en":["\\fb\\C[5](Él me está clavando la mirada......)"],
```
Valorar omitir "Él"

rayo329 commented on RemLines.json:
```diff
@@ -687,7 +687,7 @@
 
 "karryn_sight_butt_exp0_1":{
-"en":["\\fb\\C[5]\"¿¡Donde estas buscando!? ¡¡Imbécil!!\""],
+"en":["\\fb\\C[5]\"¿¡Que estas mirando!? ¡¡Imbécil!!\""],
```
"Qué" exclamativo, lleva tilde

rayo329 commented on RemLines.json:
```diff
@@ -845,17 +845,17 @@
 
 "karryn_sight_org_exp1_3":{
-"en":["\\fb\\C[5]\"Ja, ja... No te atrevas a contarle esto a nadie...\""],
+"en":["\\fb\\C[5]\"Haa haa... No te atrevas a contarle esto a nadie...\""],
```
Jadeos -> "Aah aah"

rayo329 commented on RemLines.json:
```diff
@@ -891,18 +891,18 @@
 
 "karryn_down_stamina_exp1_3":{
-"en":["\\fb\\C[5]\"H¡Ah, jaja...! ¡¡Esto aún no ha terminado...!!\""],
+"en":["\\fb\\C[5]\"Hah hahh...! ¡¡Esto aún no ha terminado...!!\""],
```
Jadeos

Aah ahh

rayo329 commented on RemLines.json:
```diff
@@ -891,18 +891,18 @@
 "karryn_down_stamina_exp1_5":{
 "en":["\\fb\\C[5]\"Urgh... no lo puedo creer...",
-	"\\fb\\C[5]Me quedé sin energía tan patéticamente...\""],
+	"\\fb\\C[5]Me quedé sin energía que patética...\""],
```
poner un punto.
"Me quedé sin energía. Qué patética

rayo329 commented on RemLines.json:
```diff
@@ -970,7 +970,7 @@
 "karryn_down_falldown_exp1_2":{
 "en":["\\fb\\C[5]\"Grr... No puedo creer que bajé la guardia de esa manera...",
-	"\\fb\\C[5]Contra algún bastardo al azar también ... !!\""],
+	"\\fb\\C[5]Contra un bastardo al azar también ... !!\""],
```
Cambiar a una frase más natural y omitir "también":

"---Contra un bastardo cualquiera ... !!"

No olvidar los signos de exclamación iniciales de la frase "¡¡No puedo creer---"

rayo329 commented on RemLines.json:
```diff
@@ -1001,7 +1001,7 @@
 
 "karryn_down_falldown_exp2_3":{
-"en":["\\fb\\C[5]\"Grr... No te limites a mirar mi ingle.......\""],
+"en":["\\fb\\C[5]\"Grr... No solo mires mi ingle.......\""],
```
Valorar cambiar a algo más natural y que mantenga el sentido de la frase original "Don't just stare at my groin"

"No te quedes mirándome la ingle"

rayo329 reviewed (commented):
Comentarios varios

rayo329 committed:
Merge branch 'develop' into feature/Remlines-dragonAge

rayo329 committed:
Merge branch 'develop' into feature/Remlines-dragonAge

Dragonage13 committed:
Actualizado desde la linea 1 hasta 5020

Actualizadas las lineas conversadas en la parte anterior de 1 a 1004.
Revisado las lineas siguientes desde la 1005 hasta la 5020

rayo329 committed:
chore: Corregido hasta la línea 3647

rayo329 committed:
Merge branch 'develop' into feature/Remlines-dragonAge

rayo329 committed:
chore: Proofread/retranslate till 5047

AuroraTheFirstLight commented on RemLines.json:
```diff
@@ -23,13 +23,13 @@
 
 "karryn_sight_exp0_1":{
 "en":["\\fb\\C[5]\"Toma tus sucios ojos,",
```
"en":["\\fb\\C[5]\"Coje tus sucios ojos,",
